python_modules/dagster/dagster_tests/daemon_sensor_tests/test_run_status_sensors.py

```diff
@@ -0,0 +1,812 @@
+import tempfile
+import time
+from contextlib import contextmanager
+
+import pendulum
+import pytest
+
+from dagster import DagsterRunStatus
+from dagster._core.events import DagsterEvent, DagsterEventType
+from dagster._core.events.log import EventLogEntry
+from dagster._core.scheduler.instigation import TickStatus
+from dagster._core.storage.event_log.base import EventRecordsFilter
+from dagster._core.test_utils import create_test_daemon_workspace, instance_for_test
+
+from .conftest import workspace_load_target
+from .test_sensor_run import (
+    evaluate_sensors,
+    failure_job,
+    failure_pipeline,
+    foo_pipeline,
+    get_sensor_executors,
+    hanging_pipeline,
+    the_pipeline,
+    validate_tick,
+    wait_for_all_runs_to_finish,
+)
+
+
+@pytest.fixture(name="instance_module_scoped", scope="module")
+def instance_module_scoped_fixture():
+    # Overridden from conftest.py, uses DefaultRunLauncher since we care about
+    # runs actually completing for run status sensors
+    with instance_for_test(
+        overrides={},
+    ) as instance:
+        yield instance
+
```

Comment:
this file just has all the run status sensor tests from the main file - both to keep the test from being unreadably massive, and also to override this fixture to actually launch the runs